src/components/coin/MemberCoinAdminBlock.tsx

```diff
@@ -506,10 +507,10 @@ const useCoinLogCollection = (filter?: { nameAndEmail?: string; title?: string;
       : undefined,
     member: filter?.nameAndEmail
       ? {
-          _or: [{ name: { _like: `%${filter.nameAndEmail}%` } }, { email: { _like: `%${filter.nameAndEmail}%` } }],
+          _or: [{ name: { _like: `% ${filter.nameAndEmail}% ` } }, { email: { _like: ` % ${filter.nameAndEmail}% ` } }],
```

Comment:
Spaces can cause the LIKE operator to produce different results.

Reply:
Oops I did not touch this part and have no idea why it turns out like that. I will definitely amend it! Thanks!